source/ProgArgs.cpp

```diff
@@ -2952,6 +2975,10 @@ void ProgArgs::setFromPropertyTreeForService(bpt::ptree& tree)
 	doS3AclVerify = tree.get<bool>(ARG_S3ACLVERIFY_LONG);
 	doS3ListObjVerify = tree.get<bool>(ARG_S3LISTOBJVERIFY_LONG);
 	doStatInline = tree.get<bool>(ARG_STATFILESINLINE_LONG);
+    doS3BucketTag = tree.get<bool>(ARG_S3_BUCKET_TAG);
```

Comment:
this patch is adding 7 new user parameters in `ProgArgs.h`, so we would need to have 7 variables also here in `setFromPropertyTreeForService()` and also 7 in `getAsPropertyTreeForService()`, so some must be missing currently in both functions